tests/integration_tests/test_assets/string/compiled_prompt_reask.txt

```diff
@@ -0,0 +1,20 @@
+
+I was given the following string, which had problems due to incorrect values.
+
+{
+  "incorrect_value": "sour cream tomata",
+  "error_message": "must be exactly two words"
+}
+
+Help me correct the incorrect values based on the given error messages.
+
+Given below is XML that describes the information to extract from this document and the tags to extract it into.
```

Comment:
For string outputs, the prompt boilerplate for the original prompt and reask prompt doesn't really need the XML schema or the few shot examples in lines 13 - 20.